README.md

```diff
@@ -60,31 +62,30 @@ git clone https://github.com/atlarge-research/opendc.git
 
 #### Set up the database
 
-Set up the database, replacing `PATH_TO_DATABASE` with where you'd like to create the SQLite database. (This will replace any file named `opendc.db` at the location `PATH_TO_DATABASE`.)
-
-```bash
-cd opendc/database
-python rebuild-database.py "PATH_TO_DATABASE"
-```
+The database can be rebuilt by using the `schema.sql` file from main opendc repository. 
```

Comment:
True, that would require adding some scripts to the main repo. Will do that later.